package.json

```diff
@@ -24,13 +24,13 @@
     "bs-platform": "^2.0.0",
     "react": "^15.4.0",
     "react-dom": "^15.4.2",
-    "react-router": "^4.1.1",
-    "react-router-dom": "^4.1.1",
+    "react-router": "^4.2.0",
+    "react-router-dom": "^4.2.2",
     "reason-react": "^0.3.0"
   },
   "peerDependencies": {
-    "react-router": "^4.1.1",
-    "react-router-dom": "^4.1.1",
+    "react-router": "^4.2.0",
+    "react-router-dom": "^4.2.2",
```

Comment:
Maybe you can leave those out, I've done this in my PR https://github.com/reasonml-community/bs-react-router/pull/7 and have a couple more changes (just to avoid conflicts 😉)

Reply:
This was just a quick update, I remove them. You can add them in PR #7